raster_api/runtime/setup.py

```diff
@@ -14,7 +14,7 @@
     "starlette-cramjam>=0.3,<0.4",
     "aws_xray_sdk>=2.6.0,<3",
     "aws-lambda-powertools>=1.18.0",
-    "starlette<0.28",
+    "python-multipart==0.0.7",
```

Comment:
Why this version? 😯

Reply:
Also, why remove starlette? 🤔

Reply:
> Also, why remove starlette? 🤔

Ah, is it because of the dependabot alert?

Reply:
Starlette version is managed by FastAPI module directly (so by titiler version), so having this here is redondant and possibly dangerous.

Reply:
@vincentsarago we're pinning the min python-multipart version and removing any pin on starlette here and letting other packages manage the version. Do you think it is also a problem to pin python-multipart?

Reply:
Oh I just re-read the full thread. I think we are good and in agreement 😅